.eslintrc

```diff
@@ -1,354 +1,127 @@
-            2
-        ],
-        "no-restricted-syntax": [
-            2,
-            "ClassDeclaration",
-            "ClassExpression"
-        ],
-        "no-spaced-func": [
-            2
-        ],
-        "no-trailing-spaces": [
-            2
-        ],
-        "no-underscore-dangle": [
-            1
-        ],
-        "no-unneeded-ternary": [
-            2
-        ],
-        "object-curly-spacing": [
-            2,
-            "always"
-        ],
-        "one-var": [
-            2,
-            "never"
-        ],
-        "operator-linebreak": [
-            2,
-            "after"
-        ],
-        "quote-props": [
-            2,
-            "as-needed"
-        ],
-        "require-jsdoc": [
-            2
-        ],
-        "semi-spacing": [
-            2,
-            { "before": false,
-              "after": true }
-        ],
-        "space-after-keywords": [
-            2,
-            "always"
-        ],
-        "space-before-blocks": [
-            2,
-            "always"
-        ],
-        "space-before-function-paren": [
-            2,
-            "never"
-        ],
-        "space-before-keywords": [
-            2,
-            "always"
-        ],
-        "space-in-parens": [
-            2,
-            "never"
-        ],
-        "space-infix-ops": [
-            2
-        ],
-        "space-return-throw-case": [
-            2
-        ],
-        "space-unary-ops": [
-            2,
-            { "words": true,
-              "nonwords": false }
-        ],
-        "spaced-comment": [
-            2,
-            "always"
-        ],
-        "arrow-parens": [
-            2,
-            "as-needed"
-        ],
-        "arrow-spacing": [
-            2,
-            { "before": true,
-              "after": true }
-        ],
-        "no-const-assign": [
-            2
-        ],
-        "no-multi-spaces": [
-            2
-        ],
-        "no-multi-str": [
-            2
-        ],
-        "no-sync": [
-            1
-        ],
-        "no-negated-condition": [
-            2
-        ],
-        "padded-blocks": [
-            2,
-            "never"
-        ]
-    },
-    "env": {
-        "node": true
-    },
-    "extends": "eslint:recommended"
-}
\ No newline at end of file
+  "rules": {
+    "no-unused-vars": [1],
+    "no-console": [1],
+    "no-cond-assign": [1],
+    "no-undef": [1],
+    "no-empty": [1],
+    "indent": [1, 2],
+    "quotes": [1, "single"],
+    "linebreak-style": [1, "unix"],
+    "semi": [2, "always"],
+    "no-extra-parens": [1],
+    "curly": [2],
+    "default-case": [2],
+    "dot-location": [2, "property"],
+    "dot-notation": [2],
+    "eqeqeq": [2, "smart"],
+    "no-caller": [2],
+    "no-else-return": [2],
+    "no-empty-label": [2],
+    "no-eval": [2],
+    "no-extend-native": [2],
+    "no-extra-bind": [2],
+    "no-floating-decimal": [2],
+    "no-implied-eval": [2],
+    "no-invalid-this": [1],
```

Comment:
This used to be an error, I'm guessing with the refactor there's alot of invalid this warnings so this was intentional to change to a warning?